torch_xla/csrc/runtime/sys_util.cpp

```diff
@@ -26,23 +26,23 @@ std::string GetEnvOrdinalPath(const char* name, const std::string& defval,
   return path;
 }
 
-std::string GetEnvOrdinalPath(const char* name, const std::string& defval,
-                              const char* ordinal_env) {
+std::string GetEnvOrdinalPath(const char* const name, const std::string& defval,
+                              const char* const ordinal_env) {
   return GetEnvOrdinalPath(name, defval, GetEnvInt(ordinal_env, -1));
 }
 
-int64_t GetEnvInt(const char* name, int64_t defval) {
-  const char* env = std::getenv(name);
+int64_t GetEnvInt(const char* const name, const int64_t defval) {
```

Comment:
Does `const` provide any value for primitive types?

Reply:
Yes, it prevents the function body from changing the values unintentionally. The code is easiest to understand if we know that a value will never change.